example/reflect/README.md

```diff
@@ -0,0 +1,129 @@
+# Running The Reflect Example
+
+## Ziti Network Setup
+### Create A Ziti Network
+You will need to have ziti executables available to run from your command line. If you do not have them, the easiest way to set yourself up is to run ziti express install.
+
+To run express install, check out the instructions [here](https://openziti.github.io/ziti/quickstarts/network/local-no-docker.html)
```

Comment:
I'd say: "run one of the quickstarts found at the landing page" and just send them to openziti.github.io